packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1167,6 +1194,13 @@ class DropdownButton<T> extends StatefulWidget {
   ///    relative to text direction.
   final AlignmentGeometry alignment;
 
+  /// Applies a borderRadius to the menu.
```

Comment:
Needs a blank line after the summary statement. And the summary statement could be slightly more illuminating.
```dart
/// Defines the corner radii of the menu's rounded rectangle shape.
/// 
```